src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/__tests__/ExposureSubmissionUITests.swift

```diff
@@ -280,7 +313,54 @@ class ENAUITests_04_ExposureSubmissionUITests: XCTestCase {
 		XCTAssertTrue(app.buttons["AppStrings.ExposureSubmission.secondaryButton"].waitForExistence(timeout: .medium))
 		app.buttons["AppStrings.ExposureSubmission.secondaryButton"].tap()
 
-		// NewSubmissionFlow: Add the test for the cancel Alert
+		app.alerts["Sie wollen keine Warnung verschicken?"].buttons["Nicht warnen"].tap()
+		
+		waitForNonExistence(of: app.buttons["AppStrings.ExposureSubmission.primaryButton"], timeout: .long)
```

Comment:
I read it (and the extension) but I don't get it. What do you want to test here?

Reply:
On Test Result Screen there is alert popup. So, I am checking the flow with the popups buttons.